[stable/prometheus-redis-exporter]Add redis authoration configuration in chart (#19447)
helm/charts: stable/prometheus-redis-exporter/README.md

| `serviceMonitor.targetLabels`  | Set of labels to transfer on the Kubernetes Service onto the target.  |             |
| `script.configmap`     | Let you run a custom lua script from a configmap. The corresponding environment variable `REDIS_EXPORTER_SCRIPT` will be set automatically ||
| `script.keyname`       | Name of the key inside configmap which contains your script ||

> | `auth.enabled`       | Specifies whether redis use authentication | `false` |

Reviewer: ```suggestion
| `auth.enabled`       | Specifies whether redis uses authentication | `false` |
```

> | `auth.secret.name`       | Name of redis existing secret (ignores redisPassword) ||

Reviewer: ```suggestion
| `auth.secret.name`       | Name of existing redis secret (ignores redisPassword) ||
```

> | `auth.secret.key`       | Key of redis existing secret ||

Reviewer: ```suggestion
| `auth.secret.key`       | Name of key containing password to be retrieved from the existing secret ||
```

> | `auth.redisPassword`       | Point the redis password when not stored in a secret ||

Reviewer: ```suggestion
| `auth.redisPassword`       | Redis password (when not stored in a secret) ||
```
